Skype/Trusted-Application-API/samples/QuickStartSamples/QuickSamplesCommon/QuickSamplesCommon.csproj

```diff
@@ -67,6 +67,10 @@
       <HintPath>..\packages\Newtonsoft.Json.8.0.3\lib\net45\Newtonsoft.Json.dll</HintPath>
       <Private>True</Private>
     </Reference>
+	<Reference Include="Microsoft.Practices.Unity, Version=4.0.0.0, Culture=neutral, PublicKeyToken=6d32ff45e0ccc69f, processorArchitecture=MSIL">
```

Comment:
Is this a new nuget dependency of QuickSamplesCommon project? If yes, please update packages.config. Use Nuget package manager to update packages.config